Configure

```diff
@@ -2179,6 +2179,16 @@ EOF
 
     # Massage the result
 
+    # If the user configured disabled-shared, we allow no shared sources
```

Comment:
typo? disabled-shared -> no-shared

Reply:
True, it's a typo, should be `disable-shared`...  That one and `no-shared` are synomyms (actually, any `no-` can be replaced with `disable-`)

Reply:
I did change it to `no-shared`, to avoid confusion :wink:

Reply:
Ok, do as you prefer. I did not know about the synonym.